CHANGELOG.md

```diff
@@ -78,6 +79,7 @@
   * `cortex_compactor_tenants_processing_failed`
 * [ENHANCEMENT] Added new experimental API endpoints: `POST /purger/delete_tenant` and `GET /purger/delete_tenant_status` for deleting all tenant data. Only works with blocks storage. Compactor removes blocks that belong to user marked for deletion. #3549 #3558
 * [ENHANCEMENT] Chunks storage: add option to use V2 signatures for S3 authentication. #3560
+* [ENHANCEMENT] HA Tracker: new limit for max number of clusters tracked for single user. Default limit is 128 clusters per user. #3668
```

Comment:
**suggestion(minor):** may be worth referencing the exact config field name here. It will help someone grepping the changelog for when it was added.
```suggestion
* [ENHANCEMENT] HA Tracker: Added new limit `ha_max_clusters` to set the max number of clusters tracked for single user. The default limit is 128 clusters per user. #3668
```